chromium_src/third_party/blink/public/mojom/webpreferences/web_preferences.mojom

```diff
@@ -21,4 +21,7 @@ struct WebPreferences {
   // try to inject JavaScript to detect <video> and <audio> tag, that are
   // dynamically attached.
   bool should_detect_media_files;
+
+  map<string, string> url_and_media_detection_scripts;
```

Comment:
We should add a comment here.

Reply:
I don't think it's a good idea to pass entire scripts in web preferences. mojom messages have size limits and this is just not a great way to transfer that kind of data